test/mockFrn.js

```diff
@@ -0,0 +1 @@
+module.exports = 1234567890
```

Comment:
Do you think these files might be broken down too much given they're not used elsewhere?

To understand the test input I have to just between many files trying understand what we're testing and it confuses my simple brain.

Reply:
It's only used once during the tests I've created but the other tests in the repo would use things like `frn` and `invoiceNumber` be it in `paymentRequests` or alike.

The idea was to:
- introduce consistency between tests
- make tests act on 'real life' objects by building them out of valid common components (similar to your `asyncapi` in `enrichment`)
- have devs spend less time building up objects to test against (which 90% of the time we're only adding the fields which are used in our functions which introduces risk of how an actual object may interact)
- have tests match our `asyncapi.yaml` because we're using 'real life' objects which should match the doc 
- can still modify these 'real life' objects for specific use-case by deleting/changing keys/values which I think adds to a level of explicitness rather than creating an `paymentRequest` object (for e.g.) which doesn't have 1 field which could go unnoticed, especially to someone just joining

possible issues:
- idea you've brought up of having to flick between different files to see how an object is made up
    - the idea around it being more explicit for changes should quash this concern as if we delete a key we dnt necessarily need to know what the object now looks like but can be sure tht deleting tht key was  paramount to how the test will perform (probs should mention somewhere in its description)
    - there is possibly an argument that this will be more implicit and less explicit when we have something like `expect(result.invoiceNumber).toBe(importedPaymentRequestObject.invoiceNumber)` right there, we don't know EXACTLY what `importedPaymentRequestObject.invoiceNumber` is rather than `expect(result.invoiceNumber).toBe(explicit)`, however, we DO know that `importedPaymentRequestObject.invoiceNumber` is a valid and 'real life' example of an `invoiceNumber` that has (should've) been fed into the function `const result = whateverFuncWeAreTesting(importedPaymentRequestObject)` therefore we know we're operating on this valid value
- cluttered and difficult to read with so many small components and objects
    - yeah, if we decide on this, we need a well defined structure in-place for common components and objects to improve readability

Reply:
I agree with the intention.

Out of all the payment services we have, this is the one with the most complicated (and duplicated) setup of tests.  So maybe this is the first step on the journey towards improving that.

We can always tweak and refine as we move forward so I'm happy to go with your approach.